### PR TITLE
Added a warning if using untrusted certificates

### PR DESCRIPTION
**Pull request by ziccardi**
# Motivation

https://issues.jboss.org/browse/AEROGEAR-3415

- [ ] Fix dependencies

**NOTE**: Dependencies will be fixed as soon as the SDK will be published since new SDK features are required (see [PR74](https://github.com/aerogear/aerogear-xamarin-sdk/pull/74)).

### CODE REVIEW

**Comment by jhellar** [comment]
Reviewed and verified for both Android and iOS.

File: Example/App.xaml.cs [hunk 1]
```diff
@@ -18,10 +25,64 @@ public App ()
...
+            {
+                string url;
+
+                MobileCoreConfiguration configuration = MobileCore.Instance.Configuration;
+
+                IEnumerator<ServiceConfiguration> serviceConfigurations = configuration.ServiceConfigurations.GetEnumerator();
+
+                if (serviceConfigurations.MoveNext())
+                {
+                    url = serviceConfigurations.Current.Url;
```
Comment (wojta) [inline Example/App.xaml.cs]: so it checks only first URL in the configurations? Is this desired behaviour?

Comment (aidenkeating) [reply Example/App.xaml.cs]: @wojta This is what we agreed to do for the other showcase apps for now

**Comment by aidenkeating** [comment]
Included in https://github.com/aerogear/xamarin-showcase-template/pull/34

**Pull request closed without merging**